### DISCUSSION TITLE
Fatal: refs/remotes/origin/HEAD cannot be resolved to branch

### DISCUSSION THREAD

**Discussion by tksuoran** [community/community#23455] [Pull Requests]
<p>I seem to be unable to make a pull request. The original issue is here <a href="https://github.com/wolfpld/tracy/issues/404" rel="noopener nofollow ugc">https://github.com/wolfpld/tracy/issues/404</a></p>
<p>I am using Git for Windows.</p>
<p>Steps:</p>
<p>I forked tracy in github and tried to apply a small fix:</p>
<ul>
<li>Fork <a href="https://github.com/wolfpld/tracy" class="inline-onebox" rel="noopener nofollow ugc">GitHub - wolfpld/tracy: C++ frame profiler</a>
</li>
<li>Clone the fork (<code>git clone git@github.com:tksuoran/tracy.git cd tracy/ git remote add wolfpld </code>)</li>
<li>Create a branch (<code>git checkout -b fix-operator-precedence</code>)</li>
<li>Edit file</li>
<li>Make a commit (<code>source git add</code> and <code>git commit</code>)</li>
<li>Create pull request by pushing (<code>git push origin --set-upstream origin fix-operator-precedence</code>)</li>
</ul>
<p>Last step gives me error:</p>
<pre><code class="lang-auto">fatal: refs/remotes/origin/HEAD cannot be resolved to branch
</code></pre>

**Comment by tksuoran** [comment, marked as answer]
<p>This was my bad, I wrote origin twice in the last command.</p>

**Reply by Kwentar** [reply]
Thank you for answering your question, you saved a lot of time people like me :)

**Reply by doublechiang** [reply]
@tksuoran You are not alone, I just made the same mistake. Thanks for your sharing.

**Comment by ettaboyle** [comment]
<p>Welcome to the forums <a class="mention" href="/tksuoran">@tksuoran</a> Thank you for closing the loop - and great to see you solved your own problem :slight_smile:  :clap:t2:</p>
